### ISSUE TITLE
move composer.json

### ISSUE THREAD

**Issue by fritzmg** [FACT-Finder-Web-Components/magento2-module#15]
Shouldn't the composer.json be moved to the project root? Otherwise this module cannot be installed and updated via composer.

**Comment by davidverholen** [comment]
yes it should. Also the composer name must be all lowercase and it should uniquely identify the magento2 integration of FF web components

**Comment by a-laurowski** [comment]
I've created a pull request with mentioned composer.json file changes + few other changes required for the composer to properly install the module
